alma/controllers/hook/DisplayRefundsHookController.php

```diff
@@ -67,7 +67,7 @@ public function run($params)
             $payment = $this->getPayment($order);
         } catch (PaymentNotFoundException $e) {
             // if we can't have the payment, log why and return null
-            Logger::instance()->debug($e->getMessage());
+            Logger::instance()->warning($e->getMessage());
```

Comment:
the exception message is a programmatic information, not a ready to log message.
you can't be sure this message is uniq
Please add a log sentence as context before the `$e->getMessage()`